plugins/include/amxconst.inc

```diff
@@ -154,6 +155,18 @@ public stock const Float:NULL_VECTOR[3];
 #define CSW_VEST            31  // Custom
 #define CSW_VESTHELM        32  // Custom
 #define CSW_SHIELDGUN       99
+#define CSW_LAST_WEAPON     CSW_P90
+
+stock const CSW_ALL_WEAPONS      = (~(1<<CSW_VEST));
```

Comment:
Yeah, moving all cstrike related consts into cstrike_const.inc and just including it would make sense.